README.adoc

```diff
@@ -153,6 +153,11 @@ $ make clean
 
 Here's how to run all available tests. All tests will check all Go packages
 except those in the `vendor/` directory.
+Make sure you have docker and docker-compose available (Probably you may consider adding your user into `docker` group using `usermod -aG docker $USER`).
```

Comment:
I'm a bit reluctant to add 'helpful' tips like 'probably you may consider...'

If I'm capable of actually considering what I'm doing, I would know all about this 'problem' already. If I'm not, then I'm just randomly running commands in my shell and not knowing why.

We could rephrase it to describe that this will run as the current user and the current user would need permissions to run commands on the docker host.. link to docker on how to achieve that for xyz os.

Maybe with a 'run docker ls' to see if you have...